UPGRADE-API-1.11.md

```diff
@@ -1,5 +1,27 @@
 # UPGRADE FROM `v1.10.X` TO `v1.11.0`
 
+1. The product images should have a proper prefix added to the path, so the images could be resolved. 
+   This is now done and response of `Product Image` resource has been extended with parameter `media_path`.
+   
+    ```diff
+        {
+            "@context": "/api/v2/contexts/ProductImage",
+            "@id": "/api/v2/shop/product-images/123",
+            "@type": "ProductImage",
+            "id": "123",
+            "type": "thumbnail",
+            "path": "uo/product.jpg",
+    +       "media_path": "/media/image/uo/product.jpg"
```

Comment:
If I see correctly, there is no more `media_path`, you are updating `path` field

Reply:
I have changed it a little now :)